src/column-layout/flexible-column-layout/index.tsx

```diff
@@ -47,9 +51,10 @@ export default function FlexibleColumnLayout({
 
   // Flattening the children allows us to "see through" React Fragments and nested arrays.
   const flattenedChildren = flattenChildren(children);
+  const Tag = (tagOverride ?? 'div') as 'div';
 
   return (
-    <div
+    <Tag
```

Comment:
do we have already have test coverage for this somewhere? perhaps in the a11y tests: would they fail if this was rendered as a `div` instead of `dl`? (if not, could it be added)

Reply:
we don't have explicit test cases for that. however, if the column layout tag was not overridden to `dl`, a11y tests would fail

I have doubts that adding this scenario brings much value. WDYT?

Reply:
seems good to me